src/client/ui/cursor/iframe-cursor.js

```diff
@@ -0,0 +1,39 @@
+import hammerhead from './../deps/hammerhead';
+import CURSOR_UI_MESSAGES from './messages';
+import { sendRequestToFrame } from '../../runner/utils/iframe';
+
+var browserUtils = hammerhead.utils.browser;
+
+//HACK: we can't get element under cursor in cross-domain iframe in IE without hide cursor.
+//Instead make 'getElement' method asynchronous, we set cursor to position by one pixel farther.
```

Comment:
No, here should be something like `instead of making asynchronous method..`.
But I'd like to rewrite the comment in the following way, please correct me:

`In all the browsers the getElementFrom point function in an iframe ignores elements from the parent frame, but in IE it doesn't and our cursor ui element overlaps the target element. So we move the cursor to a position one pixel farther to avoid this.`